src/components/NavBar.jsx

```diff
@@ -6,11 +6,11 @@ export const NavBar = (props) => {
     return ( 
         <ul {...props}>
             <li key='home'>
-                <Link className="text-sm text-gray-800 hover:text-gray-950" to="/">Home</Link>
+                <Link className=" text-[#eb1b24] hover:text-gray-950" to="/">Home</Link>
             </li>
             {topics.map((topic, index) => (
                 <li key={index}>
-                    <Link to={`/${topic.slug}`} className="text-sm text-gray-700 hover:text-gray-900">
+                    <Link to={`/${topic.slug}`} className=" text-gray-600 hover:text-gray-950 no-underline hover:underline">
```

Comment:
Nice use of Links to enable us to navigate using a url to the correct list to articles!